tests/e2e/performance/test_pvc_clone_performance.py

```diff
@@ -188,4 +204,87 @@ def test_clone_create_delete_performance(self, interface_type, pvc_size, file_si
             logger.info(f"Clone number {d['clone_num']} deletion time is {d['time']} secs.")
             logger.info(f"Clone number {d['clone_num']} deletion speed is {d['speed']} MB/sec.")
 
-        logger.info("test_clones_creation_performance finished successfully.")
+
+    @pytest.fixture
+    def base_setup_multiple_clones(self, interface_type, pvc_factory, pod_factory):
+        """
+        create resources for the test_multiple_clone_create_delete_performance
+        and calculates storage size, pvc size for the pvc and clones to be created
+        exits the test if there is not enough storage size for pvc and its multiple clones
+        Args:
+            interface_type(str): The type of the interface
+                (e.g. CephBlockPool, CephFileSystem)
+            pvc_size: Size of the created PVC
+            pvc_factory: A fixture to create new pvc
```

Comment:
It is not needed to add fixtures description when they used as function arguments